src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackNotes/FeedbackNotes.tsx

```diff
@@ -73,7 +65,7 @@ export const FeedbackNotes = ({ activity }: { activity: DatavizActivity }) => {
   const addNewNote = () => {
     appletId &&
       answerId &&
-      createAnswerNote({ appletId, answerId, activityId, note: getValues().newNote });
+      createAnswerNote({ appletId, answerId, activityId, note: getValues('newNote') });
```

Comment:
could you confirm if there is no need a translation for 'newNote'?

Reply:
yes